back/config/initializers/filter_parameter_logging.rb

```diff
@@ -8,3 +8,24 @@
 Rails.application.config.filter_parameters += %i[
   password current_password passw secret token _key crypt salt certificate otp ssn import_ideas.pdf import_ideas.xlsx
 ]
+
+# Filter out base64 encoding
+#
+# Filter only exact matches of specified parameter keys. This avoids filtering false positives.
+# E.g. `image` will filter both `image` and `imageUrl`, whereas `^image$` will only filter `image`.
+Rails.application.config.filter_parameters += [
+  /^avatar$/,
+  /^header_bg$/,
+  /^image$/,
+  /^layout_image$/,
+  /^logo$/
+]
+# Custom filter that does 2 things:
+#   1. Removes base64 image encoding from multiloc values, as our WYSIWYG editors enable the addition
+#      of (multiple) images in the string value(s) of mutlilocs, which the FE encodes as base64.
+#   2. Removes base64 encoding from the `file` parameter value, leaving other information intact (e.g. filename).
+Rails.application.config.filter_parameters << lambda do |param, value|
+  if param == 'file' || (CL2_SUPPORTED_LOCALES.include?(param.to_sym) && JSON.generate(value).include?(';base64,'))
+    value.gsub!(/;base64,[^ ]*/) { ';base64,[FILTERED]' }
```

Comment:
I'm not sure whether this is robust enough, in the sense that it might capture more than the base64 payload. But it makes me wonder whether we don't want to strip out the rich text parameters completely as well, instead of messing with them. They feel a bit large to be in the logs, and they might also expose PII that's not of a lot of value.

Reply:
It should only capture base64, (from the beginning of the base64 encoding to the next space), unless someone actually types `;base64,` in their text input, which seems unlikely.

I'm happy to go with redacting the entire value, though this would mean we lose references to filenames, etc.

Reply:
For a file, we would lose this kind of info:
```Ruby
"file":"{\"file\"=\u003e\"data:application/pdf;base64,[FILTERED]\", \"name\"=\u003e\"04-08-2021-engagement-sur-l-honneur-orange-version-anglaise.pdf\"}
```

Reply:
For now, I will go with the `gsub`.

We can always adjust over time. First, I want to see that the basic idea of removing base64 works and does not cause issues on production.